src/fastga/models/weight/mass_breakdown/b_propulsion/b1_engine_weight.py

```diff
@@ -83,18 +89,23 @@ def setup(self):
         self._engine_wrapper = BundleLoader().instantiate_component(self.options["propulsion_id"])
         self._engine_wrapper.setup(self)
 
+        self.add_input(
+            "settings:weight:propulsion:k_b1", val=np.nan, desc="Engine weight tuning factor"
+        )
```

Comment:
Same remark here

Reply:
Done in [commit](https://github.com/supaero-aircraft-design/FAST-GA/pull/187/commits/fba594d37daaff6252539d78549a553fe8e70911)